README.md

```diff
@@ -1,2 +1,27 @@
-# crypto
-Cryptographic primitives used in Polygon Miden rollup
+# Miden Crypto
+This crate contains cryptographic primitives used in Polygon Miden.
+
+## Hash
+[Hash module](./src/hash) provides a set of cryptographic hash functions which are used by Miden VM and Miden Rollup. Currently, these functions are:
+
+* [BLAKE3](https://github.com/BLAKE3-team/BLAKE3) hash function with 256-bit, 192-bit, or 160-bit output. The 192-bit and 160-bit outputs are obtained by truncating the 256-bit output of the standard BLAKE3.
+* [RPO](https://eprint.iacr.org/2022/1577) hash function with 256-bit output. This hash function is an algebraic hash function suitable for recursive STARKs.
+
```

Comment:
Don't we also support Sha2 in the Miden VM?

Reply:
In the VM yes, but the VM itself (and the rollup more broadly) does not use SHA2.